src/feltorSesol/feltor.cuh

```diff
@@ -5,6 +5,7 @@
 #include "parameters.h"
 // #include "geometry_circ.h"
 #include "dg/average.h"
+#include "dg/backend/linspace.cuh"
```

Comment:
hmm ich glaub das musst du machen ich kann in deinem pull request nichts verändern, unten der probevec ist glaub ich auch nicht deklariert, kompiliert das denn?